app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,25 @@
+class OrdersController < ApplicationController
+
+  def create
+    user = User.find(session[:user_id])
+    @order = user.orders.create(status: "Ordered",
+                                total: @cart.total_cost,
+                                submitted: DateTime.now)
+    session[:cart].map do |accessory_id, quantity|
+      @order.order_accessories.create(accessory_id: accessory_id.to_i,
+                                      order_id: @order.id,
```

Comment:
don't need this order_id here since we're calling order_accessories on an instance of order already!